thorlcr/msort/tsorts.cpp

```diff
@@ -787,6 +789,14 @@ class CThorSorter : public CSimpleInterface, implements IThorSorter, implements
         transferblocksize = TRANSFERBLOCKSIZE;
         isstable = true;
         stopping = false;
+        if (activity)
+            useTLS = activity->queryMPServer().queryTLS();
```

Comment:
I think this is always same as global queryMtls() ?
checking activity->queryMPServer() implies it might be different per activity..